tests/foreman/cli/test_contentview.py

```diff
@@ -161,27 +159,14 @@ def _get_content_view_version_lce_names_set(content_view_id, version_id):
         })['lifecycle-environments']
         return {lce['name'] for lce in lifecycle_environments}
 
-    # pylint: disable=unexpected-keyword-arg
-    def setUp(self):
-        """Tests for content-view via Hammer CLI"""
-
-        super(ContentViewTestCase, self).setUp()
-
-        if ContentViewTestCase.org is None:
-            ContentViewTestCase.org = make_org(cached=True)
-        if ContentViewTestCase.env1 is None:
-            ContentViewTestCase.env1 = make_lifecycle_environment(
-                {u'organization-id': ContentViewTestCase.org['id']})
-        if ContentViewTestCase.env2 is None:
-            ContentViewTestCase.env2 = make_lifecycle_environment(
-                {u'organization-id': ContentViewTestCase.org['id'],
-                 u'prior': ContentViewTestCase.env1['label']})
-        if ContentViewTestCase.product is None:
```

Comment:
you removed product here, but not added in setupclass, so how all statements like `repo = make_repository({'product-id': self.product['id']})` works?

Reply:
not a problem, please make changes, test them and ping us to merge